services/dumper.js

```diff
@@ -14,6 +15,13 @@ const mkdirp = P.promisify(mkdirpSync);
 
 const DEFAULT_PORT = 3310;
 
+const DEFAULT_VALUE_TYPES_TO_STRINGIFY = [
+  `${Sequelize.DataTypes.ARRAY}`,
+  `${Sequelize.DataTypes.ENUM}`,
+  `${Sequelize.DataTypes.JSONB}`,
+  `${Sequelize.DataTypes.STRING}`,
```

Comment:
Should we consider `TEXT` too? (not sure)

Reply:
Right.
Added `TEXT` and `CITEXT`.
https://sequelize.org/master/manual/model-basics.html#data-types